guides/doc-Planning_Guide/topics/Introduction.adoc

```diff
@@ -0,0 +1,195 @@
++
+Red{nbsp}Hat supports many different scripting and other frameworks, including Puppet modules. Support for these frameworks is based on the Red{nbsp}Hat Knowledgebase article https://access.redhat.com/articles/369183[How does Red{nbsp}Hat support scripting frameworks].
+
+
+[[form-Red_Hat_Satellite-Architecture_Guide-Red_Hat_Satellite_6_Supported_Usage-Pulp]]
+*Pulp*:: Pulp usage is only supported via the {ProjectServer} web UI, CLI, and API. Direct modification or interaction with Pulp's local API or database is not supported, as this can cause irreparable damage to the {ProjectName} databases.
+
+
+[[form-Red_Hat_Satellite-Architecture_Guide-Red_Hat_Satellite_6_Supported_Usage-Foreman]]
+*Foreman*:: Foreman can be extended using plug-ins, but only plug-ins packaged with {ProjectName} are supported. Red{nbsp}Hat does not support plug-ins in the {ProjectName} Optional repository.
++
+{ProjectName} also includes components, configuration and functionality to provision and configure operating systems other than {RHEL}. While these features are included and can be employed, Red{nbsp}Hat supports their usage for {RHEL}.
+
+
+[[form-Red_Hat_Satellite-Architecture_Guide-Red_Hat_Satellite_6_Supported_Usage-Candlepin]]
+*Candlepin*:: The only supported methods of using Candlepin are through the {ProjectName} web UI, CLI, and API. Red{nbsp}Hat does not support direct interaction with Candlepin, its local API or database, as this can cause irreparable damage to the {ProjectName} databases.
+
+
+[[form-Red_Hat_Satellite-Architecture_Guide-Red_Hat_Satellite_6_Supported_Usage-Embedded_Tomcat_Application_Server]]
+*Embedded Tomcat Application Server*:: The only supported methods of using the embedded Tomcat application server are through the {ProjectName} web UI, API, and database. Red{nbsp}Hat does not support direct interaction with the embedded Tomcat application server's local API or database.
+
+[[sect-Red_Hat_Satellite-Architecture_Guide-Red_Hat_Satellite_6_Supported_Client_Architectures]]
+=== Supported Client Architectures
+
+include::Content_Management_Support.adoc[]
+
+include::Host_Provisioning_Support.adoc[]
+
+include::Configuration_Management_Support.adoc[]
+
+[NOTE]
+====
+Usage of all {ProjectName} components is supported within the context of {ProjectName} only. Third-party usage of any components falls beyond supported usage.
+====
+
+endif::[]
+
+ifeval::["{build}" != "satellite"]
+[[Supported_Operating_Systems_and_Architectures]]
+=== Supported Operating Systems and Architectures
+
+[[Foreman_Server_Operating_System]]
+==== {ProjectServer} Operating System
+
+{Project} has packages for CentOS 7 and clones, and Debian and clones. CentOS is the recommended platform for the deployment.
```

Comment:
> CentOS is the recommended platform for the deployment.

Maybe "Katello plugin packages, which provides content management capabilities, are only available for CentOS.